dirsrvtests/tests/suites/plugins/dna_interval_test.py

```diff
@@ -82,3 +94,98 @@ def test_dna_interval(topology_st):
     log.info("Test DNA interval assignment is working...")
     user.replace('uidNumber', '-1')
     assert user.get_attr_val_utf8_l('uidNumber') == '20'
+
+
+def test_dna_max_value(topology_st, dna_plugin):
+    """Test the dna max value works with dna interval
+
+    :id: cc979ea8-3cd0-4d52-af35-9cea7cf8cb5f
+    :setup: Standalone Instance
+    :steps:
+        1. Set dnaMaxValue, dnaNextValue, dnaInterval values to 100, 90, 50 respectively
+        2. Create user that trigger DNA to assign a value
+        3. Try to make an update to entry that triggers DNA again
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Operation should fail with OPERATIONS_ERROR
+    """
+    log.info("Make the config changes needed to test dnaMaxValue")
+    dna_plugin.replace_many(('dnaMaxValue', '100'), ('dnaNextValue', '90'), ('dnaInterval', '50'))
+    # Create user2
+    users = UserAccounts(topology_st.standalone, DEFAULT_SUFFIX)
+    log.info('Adding user2')
+    user = users.create(properties={
+        'sn': 'interval2',
+        'cn': 'interval2',
+        'uid': 'interval2',
+        'uidNumber': '-1',  # Magic regen value
+        'gidNumber': '222',
+        'givenname': 'interval2',
+        'homePhone': '0861234567',
+        'carLicense': '131D16674',
+        'mail': 'interval2@whereever.com',
+        'homeDirectory': '/home/interval2'})
+
+    # Verify DNA works
+    assert user.get_attr_val_utf8_l('uidNumber') == '90'
```

Comment:
Additional checks to make sure that DNA is working correctly. It can be removed if not needed